### DISCUSSION TITLE
Is there a memory limit to the GUI?

### DISCUSSION THREAD

**Discussion by FateDecider** [qcha41/usit#29] [General]
I am acquiring multiple frames of data (to average the signals) from an oscilloscope (Picoscope brand, to be precise).

I seem to be able to work with up to 600 frames very easily on Jupyter notebook (without recasting the code into OOP style to fit autolab's architecture).

But, somehow, this doesn't work well on the GUI and the GUI closes with code 3221225477 (0xc0000005) when I acquire more than 10 frames. The GUI works comfortably at about 6 frames, so it is definitely not a syntax error. I would still prefer to average hundreds of frames on the GUI if possible.

**Comment by Python-simulation** [comment]
The error code stands for 'STATUS_ACCESS_VIOLATION' which should not be linked to memory issue but more likely to a variable reading issue. Could you details a bit how you tried to acquire the 600 frames or better send the scan config file so I can try to replicate it with a dummy driver?

**Reply by FateDecider** [reply]
I was talking with the team at Picoscope, and I can confirm this is an equipment-specific issue. We can close this as resolved, thank you!
